config/config.default.yaml

```
@@ -482,6 +482,7 @@ sector:
     limited_heat_sources:
       geothermal:
         constant_temperature_celsius: 65
+        ignore_missing_regions: false
```

```suggestion
        ignore_missing_regions: true
```
The default should be more lenient, I would say. Otherwise, the standard config will fail, wouldn't it?